docs/Contributing-to-Tachyon.md

```diff
@@ -7,11 +7,12 @@ group: Development
 
 Thank you for your interest in Tachyon! We greatly appreciate any new features or fixes.
 
+### toc
 * [Submitting Code](#submitting-code)
```

Comment:
You're right, this was a previously existing table of contents but I fixed the indentation